torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -377,6 +388,26 @@ void UpdateOutputTypeByONNXProto(
   }
 }
 
+void FetchBlockInputMetadataFromParent(Block* b) {
+  auto n = b->owningNode();
+  if (nullptr != n && n->kind() == ::c10::onnx::Loop) {
+    // Copy node input metadata to subgraph input.
+    for (size_t i = 0; i < n->inputs().size(); ++i) {
```

Comment:
auto?

Reply:
i remember auto triggers clang tidy error, that why switched to size_t